doc/onboarding-extras.md

```diff
@@ -1,26 +1,24 @@
 ## Who to CC in issues
 
-* `lib/buffer`: @trevnorris
-* `lib/child_process`: @cjihrig, @bnoordhuis, @piscisaereus
-* `lib/cluster`: @cjihrig, @bnoordhuis, @piscisaereus
-* `lib/{crypto,tls,https}`: @indutny, @shigeki, @nodejs/crypto
-* `lib/domains`: @misterdjules
-* `lib/{_}http{*}`: @indutny, @bnoordhuis, @nodejs/http
-* `lib/net`: @indutny, @bnoordhuis, @piscisaereus, @chrisdickinson, @nodejs/streams
-* `lib/{_}stream{s|*}`: @nodejs/streams
-* `lib/repl`: @fishrock123
-* `lib/timers`: @fishrock123, @misterdjules
-* `lib/zlib`: @indutny, @bnoordhuis
-
-* `src/async-wrap.*`: @trevnorris
-* `src/node_crypto.*`: @indutny, @shigeki, @nodejs/crypto
-
-* `test/*`: @nodejs/testing, @trott
-
-* `tools/eslint`, `.eslintrc`: @silverwind, @trott
-
-* upgrading v8: @bnoordhuis / @targos / @ofrobots
-* upgrading npm: @thealphanerd, @fishrock123
+| subsystem | maintainers |
+| --- | --- |
+| `lib/buffer` | @trevnorris |
+| `lib/child_process` | @cjihrig, @bnoordhuis |
+| `lib/cluster` | @cjihrig, @bnoordhuis |
+| `lib/{crypto,tls,https}` | @nodejs/crypto |
+| `lib/domains` | @misterdjules |
+| `lib/{_}http{*}` | @indutny, @bnoordhuis, @mscdex, @nodejs/http |
+| `lib/net` | @indutny, @bnoordhuis, @nodejs/streams |
+| `lib/{_}stream{s|*}` | @nodejs/streams |
+| `lib/repl` | @fishrock123 |
+| `lib/timers` | @fishrock123, @misterdjules |
+| `lib/zlib` | @indutny, @bnoordhuis |
+| `src/async-wrap.*` | @trevnorris |
+| `src/node_crypto.*` | @nodejs/crypto |
+| `test/*` | @nodejs/testing, @trott |
+| `tools/eslint`, `.eslintrc` | @silverwind, @trott |
+| upgrading v8 | @bnoordhuis, @targos, @ofrobots |
+| upgrading npm | @thealphanerd, @fishrock123 |
```

Comment:
For some areas, yes.

For other parts of core, I think this is still correct until the knowledge becomes more distilled and distributed as more people learn the codebase, and while that is something we should work towards it doesn't necessarily reflect reality.